services/121-service/src/program-financial-service-provider-configurations/dtos/update-program-financial-service-provider-configuration.dto.ts

```diff
@@ -0,0 +1,25 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
+import {
+  IsArray,
+  IsDefined,
+  IsNotEmpty,
+  IsOptional,
+  ValidateNested,
+} from 'class-validator';
+
+import { CreateProgramFinancialServiceProviderConfigurationPropertyDto } from '@121-service/src/program-financial-service-provider-configurations/dtos/create-program-financial-service-provider-configuration-property.dto';
+import { LocalizedString } from '@121-service/src/shared/types/localized-string.type';
+
+export class UpdateProgramFinancialServiceProviderConfigurationDto {
+  @ApiProperty({ example: { en: 'FSP display name' } })
+  @IsNotEmpty()
+  public readonly label: LocalizedString;
+
+  @IsArray()
```

Comment:
missing @ApiProperty (with example value)

Reply:
Ah yes I left it empty to reduce the change of people accidently overwriting all the properties. I update the documentation of the endpoint now to clarify this: 
 ```
 @ApiOperation({
    summary:
      'Update a Financial Service Provider Configuration for a Program. Can only update the label and properties. Posting an empty array of properties will delete all properties. The formatting of the properties can found at: /programs/{programId}/financial-service-provider-configurations/{name}/properties',
  })
```